### DISCUSSION TITLE
basic geo information in the recording

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1426] [users: ideas for new features]
We want to include some coordinates and perhaps even some use of the ipad's GPS.

**Comment by urapadmin** [comment]
@bestock, @luizaogs: 
I wonder if we should allow projects to enter full coordinates in elevations for all 10 values: 
- We would switch to inputting the elevation in a special coordinate popup (x,y,z and mapping)
- the main locus form would show only the z (as an information)
- Projects that don't want to use full coordinations could just enter the Z values (they would still have to do that in the popup, though, so overall input gets a bit less comfortable)
- the coordinates could be filled in via the GPS module of the ipad
- Parker's central coordinate would be covered (he would input z values only in the others)
- Leslie's wish to use full coordinates would be covered

see #1380

**Reply by urapadmin** [reply]
we have decided against this.

**Comment by luizaogs** [comment]
As long as it's configurable by project, as you say above, that seems to make sense to me.

**Comment by urapadmin** [comment]
> As long as it's configurable by project, as you say above, that seems to make sense to me.

I wasn't thinking so much of configuration but just using the Z coordinate if a project does not care for full coordinates. Of course, we could also make the x and y invisible on a config key.

**Comment by urapadmin** [comment]
Question remains: Do we want to input coordinates somewhere else and use the device's GPS for it? I think I'll turn this into a discussion.

**Comment by urapadmin** [comment]
@nparkervan: You might want to join this discussion

**Comment by nparkervan** [comment]
Hey all -- we enter the z values for loci in meters above seal level as is. If you want to add x, y and z, you could, but with our workflow those would be more useful as attributes of the unit corners
